File: dto-mapping-benchmark/src/main/java/com/example/dtomappingbenchmark/v3/repository/PostRepository.java
package com.example.dtomappingbenchmark.v3.repository;

import com.example.dtomappingbenchmark.v3.entity.Post;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository("postRepositoryV3")
public interface PostRepository extends JpaRepository<Post, Long> {
}

File: object-mapping-benchmark-v5/src/main/kotlin/yu/selab/cj/objectmappingbenchmarkv5/post/repository/PostRepository.kt
package yu.selab.cj.objectmappingbenchmarkv5.post.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import yu.selab.cj.objectmappingbenchmarkv5.post.entity.Post


@Repository
interface PostRepository : JpaRepository<Post, Long> {
}